packages/medusa/src/services/order-edit-item-change.ts

```diff
@@ -125,7 +125,7 @@ export default class OrderEditItemChangeService extends TransactionBaseService {
 
       const lineItemServiceTx = this.lineItemService_.withTransaction(manager)
       await Promise.all([
-        ...lineItemIdsToRemove.map((id) => lineItemServiceTx.delete(id)),
+        ...lineItemIdsToRemove.map(async (id) => lineItemServiceTx.delete(id)),
```

Comment:
question: the delete return a promise, why do we need the async keyword here?

Reply:
because of our styleguide 😄 specifically this rule: @typescript-eslint/promise-function-async

Reply:
Ok if this is the rule then… i am really not use to do that with a map that just return a promise 😂